lib/rules/color-named/generateColorFuncs.js

```diff
@@ -102,11 +102,7 @@ function rgb2hsl(r, g, b) {
 
 	l = (M + m) / 2;
 
-	if (d === 0) {
-		s = 0;
-	} else {
-		s = d / (1 - Math.abs(2 * l - 1));
-	}
+	s = d === 0 ? 0 : d / (1 - Math.abs(2 * l - 1));
```

Comment:
I would rather keep if statement here, because this line has so many symbols, it's very hard for a human to understand it.

Reply:
This whole PR is automatic and I'd rather keep it that way.